.github/workflows/tests.yml

```diff
@@ -128,8 +134,12 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python_version: ["3.6", "3.7", "3.8"]
+        python_version: ["3.6", "3.7", "3.8", "3.9"]
         optional_libraries: ["minimal", "optional"]
+        exclude:
+          # excludes python 3.9 until Koalas support released
```

Comment:
if we exclude the optional tests entirely, we won't test dask on 3.9.  

we could try adding a step like this
```
pip uninstall -r koalas-requirements.txt -y
```